kotlin/viewregistry-android/src/main/res/layout-sw600dp-land/panel_container.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
```

Comment:
Why ConstraintLayout rather than FrameLayout?

Reply:
Because I need the ability to show the base view behind the modals on tablets. Note the code in PanelContainer that calculates the size of `panelBody` based on the screen size.